### PR TITLE
Add support to reset very low settings + another updates

### PR DESCRIPTION
**Pull request by Tiagoquix**
* Added a new module: htmlmotd -- controls HTML MOTDs. Default is off for all presets.

* Added support to reset Very Low preset's settings: some settings are saved in the ``config.cfg`` file and some of them are personal preferences. I tried moving the maximum amount of them to the main ``comfig.cfg`` file, but some of them I put in another file called ``very_low_reset.cfg`` that can be executed using ``very_low_reset``. I also added new instructions in the Quick Fixes page on the docs.

* ``tf_sniper_fullcharge_bell`` is now enabled by default in the main ``comfig.cfg`` file.

@mastercoms

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -21,7 +21,7 @@
 net_maxpacketdrop 0 // Use all packets
 //net_maxpacketdrop 5000 // Use default packet drop threshold
 cl_timeout 60 // Max client timeout
-//cl_smooth 1 // Do not instantly correct prediction errors, as they are predominantly small enough to be smoothed over with neglible error and has less distracting jitters in view
+cl_smooth 1 // Do not instantly correct prediction errors, as they are predominantly small enough to be smoothed over with neglible error and has less distracting jitters in view
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: This is on by default, and cannot be saved.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 2]
```diff
@@ -941,7 +941,7 @@ alias opengl opengl_default
 
 //cl_autoreload 1 // Auto reload clip
 //cl_autorezoom 1 // Automatically rezoom the Sniper Rifle
-//tf_sniper_fullcharge_bell 1 // Bell sound when the Sniper Rifle is 100% charged
+tf_sniper_fullcharge_bell 1 // Bell sound when the Sniper Rifle is 100% charged
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: People complained about this, no point in enabling it either

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 3]
```diff
@@ -1519,7 +1527,7 @@ snd_legacy_surround 0 // Turn off legacy surround sound pathway
 // --------------------
 // Optimizations with no noticeable effect on the listener
 
-//snd_cull_duplicates 0 // Do not skip playing duplicate sounds
+snd_cull_duplicates 0 // Do not skip playing duplicate sounds
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Default is 0, not saved

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 4]
```diff
@@ -1576,7 +1584,7 @@ alias logo logo_on
 // ----------------
 // Keyboard adjustments
 
-//in_usekeyboardsampletime 1 // Determine key time from smoothed frametime for keyboard look only
+in_usekeyboardsampletime 1 // Determine key time from smoothed frametime for keyboard look only
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Default is 1, not saved

File: docs/customization/modules.md [hunk 5]
```diff
@@ -763,6 +763,20 @@ Default setting: **`console=on`** (all presets, except Very Low).
 * **`console=off`**: Disables developer console.
 * **`console=on`**: Enables developer console.
 
+### HTML MOTD
+
+Controls HyperText Markup Language Message Of The Day (HTML MOTD). This includes ads, music player, rules, fancy server intros, etc.
```
Comment (mastercoms) [inline docs/customization/modules.md]: Just say HTML MOTD or HTML Message of the Day.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 6]
```diff
@@ -1201,6 +1208,7 @@ cl_mvm_wave_status_visible_during_wave 1 // MvM wave information during the wave
 //con_enable 1 // Enable the developer console
 con_nprint_bgalpha 100 // Console translucency (0 to 255) for special debug messages (nprint)
 //con_nprint_bgalpha 0 // Disable drawing box around nprint
+con_drawnotify 1 // Draw notification area (for taking screenshots)
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Default is 1, not saved

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 7]
```diff
@@ -1014,6 +1014,7 @@ tf_scoreboard_ping_as_text 1 // Use text for ping in the scoreboard
 //cl_mainmenu_safemode 1 // Dev safe mode for main menu
 //tf_queue_spinner_color 0 // Orange queue TF logo spinner
 //tf_queue_spinner_color 1 // Blue queue TF logo spinner
+tf_hud_notification_duration 3 // How long notifications appear on screen
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Default is 3, not saved

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by Tiagoquix** [comment]
All done @mastercoms

File: config/mastercomfig/cfg/comfig/very_low_reset.cfg [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+cl_show_market_data_on_items 1
+cl_showbackpackrarities 0
+cl_spec_carrieditems 1
+hud_freezecamhide 0
+r_drawviewmodel 1
+tf_contract_competitive_show 2
+tf_contract_progress_show 1
+tf_hud_target_id_disable_floating_health 0
+tf_simple_disguise_menu 0
+tf_sniper_fullcharge_bell 0
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/very_low_reset.cfg:10]: This should 1, not 0.

File: config/mastercomfig/cfg/comfig/very_low_reset.cfg [hunk 9]
```diff
@@ -0,0 +1,14 @@
+cl_crosshairalpha 200
+cl_hud_minmode 0
+cl_show_market_data_on_items 1
+cl_showbackpackrarities 0
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/very_low_reset.cfg]: Very Low forces the default of 0, no need to reset it.

File: config/mastercomfig/cfg/comfig/very_low_reset.cfg [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+cl_spec_carrieditems 1
+hud_freezecamhide 0
+r_drawviewmodel 1
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/very_low_reset.cfg]: Not saved

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: config/mastercomfig/cfg/comfig/very_low_reset.cfg (hunk 8 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/very_low_reset.cfg:10]: Its default value is 0. That's why cl_crosshairalpha is 200 and not 255, for example.

Comment (mastercoms) [reply config/mastercomfig/cfg/comfig/very_low_reset.cfg:10]: Sorry, was looking in the wrong place for the default

**Review by mastercoms — APPROVED** [review]

**Pull request merged**